config/identity/config.yaml

```diff
@@ -203,6 +203,11 @@ ci-issuer-metadata:
       # for cases that the claimed data doesn't exist.
       # platform_url: Codefresh platform url
       platform_url: "https://g.codefresh.io"
+      scm_repo_url: ""
+      scm_ref: ""
+      runner_environment: ""
+      account_name: ""
```

Comment:
@ilia-medvedev-codefresh Could you confirm if `account_name` and `pipeline_name` are optional? Given those are a part of the [SAN](https://github.com/sigstore/fulcio/blob/77325fa27326bb1e6886811def8856f431f1998d/pkg/identity/codefresh/principal.go#L148), can we require they are present?

Reply:
@haydentherapper Those are present by default in any token provided the Codefresh OIDC provider, so yes.

Reply:
Thanks, marked as required.